### PR TITLE
OCP 4.17: Installation document update

### PR DESCRIPTION
**Pull request by alishaIBM**
Version(s): 4.17

Issue: https://jsw.ibm.com/browse/POCPQE-82

Link to docs preview:
- https://81279--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_ibm_power/installing-ibm-power.html#minimum-ibm-power-system-requirements_installing-ibm-power
- https://81279--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_ibm_power/installing-restricted-networks-ibm-power.html#minimum-ibm-power-system-requirements_installing-restricted-networks-ibm-power

QE review:
- [x] QE has approved this change.

Additional information:
https://docs.google.com/document/d/13zlqG7EPWz3kkVAvv-VxTjgfxoTzmxTdTRRXGiukXrA/edit

### CODE REVIEW

**Review by prb112 — APPROVED** [review]
/lgtm

File: modules/minimum-ibm-power-system-requirements.adoc [hunk 1]
```diff
@@ -35,7 +35,7 @@ On your {ibm-power-name} instance, set up:
 [discrete]
 == Disk storage for the {ibm-power-title} guest virtual machines
 
-* Local storage, or storage provisioned by the Virtual I/O Server using vSCSI, NPIV (N-Port ID Virtualization) or SSP (shared storage pools)
+* Local storage, or storage provisioned by the Virtual I/O Server using vSCSI, NPIV (N-Port ID Virtualization), Fibre Channel or SSP (shared storage pools)
```
Comment (ShaunaDiaz) [inline modules/minimum-ibm-power-system-requirements.adoc:38]:
```suggestion
* Local storage, or storage provisioned by the Virtual I/O Server using vSCSI, NPIV (N-Port ID Virtualization), Fibre Channel, or SSP (shared storage pools)
```
https://www.ibm.com/docs/en/ibm-style?topic=punctuation-commas#commas-between-items-in-a-series

**Comment by alishaIBM** [comment]
@ShaunaDiaz thank you for the review. I have incorporated the changes.

/label merge-review-needed

**Pull request merged**